master/security/service-accounts.md

```diff
@@ -87,6 +88,32 @@ spec:
   selector: 'app == "db"'
 ```
 
+#### Use Kubernetes RBAC to control service account label assignment
+
+Network policies can be applied to endpoints using selectors that match labels on either the endpoint itself, the endpoint's namespace, or the endpoint's service account. By specifying selectors based on the endpoint's service account we can employ Kubernetes RBAC to control which users can assign labels to service accounts. That group may be separate from the group of users who are allowed to deploy pods.
+
+In the following example, pods with an **intern** service account will only be allowed to communicate with other pods with service accounts labeled `role: intern`.
```

Comment:
In the following example, pods with an **intern** service account are allowed to communicate only with pods with service accounts labeled `role: intern`.

Reply:
Fixed